contract/src/lib.rs

```diff
@@ -228,8 +166,8 @@ impl Contract {
     /// - Many cases
     #[pause(except(roles(Role::UnrestrictedSubmitBlocks)))]
     pub fn submit_block_header(&mut self, block_header: Header) {
-        // Chainwork is validated inside block_header structure (other consistency checks too)
-        let prev_blockhash = block_header.prev_blockhash.to_string();
+        // // Chainwork is validated inside block_header structure (other consistency checks too)
+        // let prev_blockhash = block_header.prev_block_hash.to_string();
```

Comment:
Remove this comments

Reply:
afffd93a42b425eaf75acb1a438e5f79f5c94c17